src/js/puzzles.js

```diff
@@ -314,10 +314,29 @@ var puzzlesArray = [
   word: 'cards',
   hint: 'unit of distance',
   answer: 'yards'
+},
+{
+  id: 46,
+  sentence: 'I left my boat at the rock',
+  word: 'rock',
+  hint: 'on water',
+  answer: 'dock'
+},
+{
+  id: 47,
+  sentence: 'Everytime it gains, it pours',
+  word: 'gains',
+  hint: 'weather',
+  answer: 'rains'
+},
+{
+  id: 48,
+  sentence: 'Are you sure, this will be sun?',
+  word: 'sun',
+  hint: 'Having the best time of my life',
```

Comment:
Superlative here seems a little strong. Maybe something like "having a good time".

Reply:
About to send pr with changes